test/e2e/upgrade_test.go

```diff
@@ -45,7 +45,10 @@ func testUpgradeVersion(t *testing.T) {
 	testCtx, err := NewTestContext(t)
 	require.NoError(t, err)
 
+	startTime := time.Now()
 	err = testCtx.waitForWindowsNodes(gc.numberOfNodes, true, false, true)
+	endTime := time.Now()
+	log.Printf("%v time is required to configure %v nodes", endTime.Sub(startTime), gc.numberOfNodes)
```

Comment:
Might be useful to add this in `waitForWindowsNodes` itself, so that we can see the time in all cases.

Reply:
Should I squash the second commit ?

Reply:
Yes, please. I think none of it is throw away work.